app/src/androidTest/java/at/sw21_tug/team_25/expirydates/HomeFragmentMapTest.kt

```diff
@@ -0,0 +1,38 @@
+package at.sw21_tug.team_25.expirydates
+
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.filters.LargeTest
+import androidx.test.rule.ActivityTestRule
+import androidx.test.rule.GrantPermissionRule
+import androidx.test.runner.AndroidJUnit4
+import org.hamcrest.Matchers.allOf
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@LargeTest
+@RunWith(AndroidJUnit4::class)
+class HomeFragmentMapTest {
+
+    @Rule
+    @JvmField
+    var mActivityTestRule = ActivityTestRule(MainActivity::class.java)
+
+    @Rule
+    @JvmField
+    var mGrantPermissionRule =
+            GrantPermissionRule.grant(
+                    "android.permission.ACCESS_FINE_LOCATION")
+
+    @Test
+    fun homeFragmentMapTest() {
+        val view = onView(
+allOf(withContentDescription("Google Map"),
```

Comment:
could you please format the file? I know it was generated. Code->Reformat File ....

Reply:
Yeah sure, good point!